### PR TITLE
Added _validate_collection and validate_custom_attributes methods

### PR DESCRIPTION
**Pull request by iakovdudinsky**

### CODE REVIEW

File: deker/validators.py [hunk 1]
```diff
@@ -134,6 +134,46 @@ def process_attributes(
...
+) -> dict:
+    """Validate custom attributes over schema.
+
+    :param schema: ArraySchema or VArraySchema instance
+    :param dimensions: tuple of (V)Array dimensions
+    :param primary_attributes: (V)Array primary attributes
+    :param custom_attributes: old custom attributes
+    :param attributes: new custom attributes to validate
+    """
+    from deker.schemas import TimeDimensionSchema
```
Comment (gurich92) [inline deker/validators.py:152]: Why import it here and not in the start?

Comment (iakovdudinsky) [reply deker/validators.py:152]: Circular Import

File: deker/validators.py [hunk 2]
```diff
@@ -42,12 +42,19 @@ def process_time_dimension_attrs(attributes: dict, attr_name: str) -> datetime.d
     return time_attribute
 
 
-def __process_attrs(
+def __process_attributes_types(
     attrs_schema: Tuple["AttributeSchema", ...],
     attributes: dict,
```
[the remark this answers is not in the document]
Comment (iakovdudinsky) [reply deker/validators.py:47]: for attr in attrs_schema

[the remark this answers is not in the document]
Comment (iakovdudinsky) [reply deker/validators.py:47]: if attr.name not in attributes:

**Review by matveyvarg — APPROVED** [review]

**Pull request merged**